### ISSUE TITLE
Error 500 returns HTML and not a JSON body

### ISSUE THREAD

**Issue by mvandenbrink** [stargate/stargate#740]
I was testing with the REST and Document API ( `{{baseUrl}}/v1/keyspaces/test/tables/filings/rows` ) on a table that was created with CQL. When I send a request with the REST API, the response is an JSON.

```
{
    "count": 0,
    "rows": []
}
```

When I send a request with the Document API ( `{{baseUrl}}/v2/namespaces/test/collections/filings` ), the response is an Error 500 Internal Server Error, but the response body is HTML and not JSON.

```
<html>

<head>
	<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
	<title>Error 500 Internal Server Error</title>
</head>

<body>
	<h2>HTTP ERROR 500 Internal Server Error</h2>
	<table>
		<tr>
			<th>URI:</th>
			<td>/v2/namespaces/test/collections/filings</td>
		</tr>
		<tr>
			<th>STATUS:</th>
			<td>500</td>
		</tr>
		<tr>
			<th>MESSAGE:</th>
			<td>Internal Server Error</td>
		</tr>
		<tr>
			<th>SERVLET:</th>
			<td>io.dropwizard.jersey.setup.JerseyServletContainer-df6f662</td>
		</tr>
	</table>

</body>

</html>
```

In this way the body can't be inspected in a proper way. Please consider to use always a JSON body and never a HTML body.

**Comment by eribeiro** [comment]
Hi @mvandenbrink, once #630 is merged all error messages the Documents API call will return JSON documents. Btw, `{{baseUrl}}/v2/namespaces/test/collections/filings` returns 500 because the DocsAPI has its own internal structure, particularly a collection table to support documents. IIRC, SG version 1.0.14 doesn't return a HTML page, but I didn't have time to test it again.

**Comment by eribeiro** [comment]
Hi @dougwettlaufer, I __guess__ this issue was solved after the merging of #630. Please @mvandenbrink, would you let me know if there's any further action necessary to address it otherwise? Thanks! :)

**Comment by mvandenbrink** [comment]
Hi @eribeiro, thanks for solving the issue. I used version 1.0.14 and I can confirm that response is now correct JSON, even if I use a not existing table or keyspace. I will close the issue.
